src/core/config/wakeup.h

```diff
@@ -117,6 +117,15 @@
 #define OPENTHREAD_CONFIG_WED_RECEIVE_TIME_AFTER 500
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_WAKEUP_FRAME_TX_CCA
+ *
+ * Whether wake-up frames are transmitted after performing CCA.
+ */
+#ifndef OPENTHREAD_CONFIG_WAKEUP_FRAME_TX_CCA
+#define OPENTHREAD_CONFIG_WAKEUP_FRAME_TX_CCA 1
```

Comment:
nit: How about to change the name to `OPENTHREAD_CONFIG_WAKEUP_FRAME_TX_CCA_ENABLED`?